src/DotnetWebApiBench.DataAccess/NorthwindDatabaseContext.cs

```diff
@@ -23,22 +23,33 @@
 using DotnetWebApiBench.DataAccess.Entity;
 using Microsoft.EntityFrameworkCore;
 using System;
+using System.Diagnostics;
 
 namespace DotnetWebApiBench.DataAccess
 {
     public partial class NorthwindDatabaseContext : DbContext
     {
+        private string serverType;
         public NorthwindDatabaseContext()
         {
             this.Database.SetCommandTimeout(TimeSpan.FromMinutes(2));
             this.Database.ExecuteSqlRaw("PRAGMA journal_mode=WAL");
         }
 
-        public NorthwindDatabaseContext(DbContextOptions<NorthwindDatabaseContext> options)
+        public NorthwindDatabaseContext(DbContextOptions<NorthwindDatabaseContext> options, string serverType)
             : base(options)
         {
+            this.serverType = serverType;
             this.Database.SetCommandTimeout(TimeSpan.FromMinutes(2));
-            this.Database.ExecuteSqlRaw("PRAGMA journal_mode=WAL");
+            if (serverType != null && serverType.Equals("SQLServer", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
I suggest to revert a condition - only execute that PRAGMA command if the database type is set to SQLite.

Reply:
SQLite is the default database for these benchmark if you won't specify db-type startup argument.

Reply:
Yes, I know - I was referring only to the readability of the code - after adding an enum with Database type it would be better to change the condition so it adds the pragma command only if the type of the database from the enum is set to SQLite.

Reply:
changes applied